src/components/new-conversation-dialog/create-room/index.js

```diff
@@ -39,26 +40,31 @@ const CreateRoom = ({
   onCreateRoom,
 }) => {
   useEffect(() => {
-    onSearchMembers()
```

Comment:
Take it back please, for now, I implemented showing members list on click to input field in another branch

Reply:
sorry, yes!